src/Atoms/FooterProfileButton/FooterProfileButton.js

```diff
@@ -0,0 +1,38 @@
+import propTypes from 'prop-types';
+import { FaUserCircle } from 'react-icons/fa';
+import { MdSettings } from 'react-icons/md';
+import { useNavigate } from 'react-router-dom';
+
+import { Div, FooterProfileButtonStyled } from './FooterProfileButtonStyled';
+
+const FooterProfileButton = ({ withSettings }) => {
+  const navigate = useNavigate();
+  function clickHandler() {
+    {
+      withSettings ? navigate('./settings') : navigate('./user');
+    }
+  }
+  return (
+    <FooterProfileButtonStyled onClick={clickHandler}>
+      {withSettings ? (
+        <>
+          <MdSettings />
+          <Div> Ustawienia </Div>
+        </>
+      ) : (
+        <>
+          <FaUserCircle />
+          <Div> Twój profil </Div>
+        </>
+      )}
+    </FooterProfileButtonStyled>
+  );
+};
+
+FooterProfileButton.propTypes = {
+  text: propTypes.string,
+  active: propTypes.string,
+  withSettings: propTypes.bool,
```

Comment:
think only withSettings is needed. Still not my scope problem :)

Reply:
Well, I think its best if you make a decision on how to go about it. Most probably the person who knows most about the component is the one who wrote it, so it could be this particular person to update it. I think it could be a general issue that all of the team members would go through their components to check if the values are described correctly in propTypes